Memorize/ContentView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ContentView.swift
+//  Memorize
+//
+//  Created by Admin on 22.01.2023.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @ObservedObject var viewModel: EmojiMemorize
+    
+    var body: some View {
+        VStack{
+            Text(EmojiMemorize.thema.name)
+                .font(.title)
+                .fontWeight(.bold)
+                
+            ScrollView{
```

Comment:
Please make space after {

Reply:
I would highly recommend using swiftlint,  you can find it on GitHub, I gives you a warning when your code style is not matching